glide-core/src/protobuf/redis_request.proto

```diff
@@ -255,13 +264,21 @@ message Transaction {
     repeated Command commands = 1;
 }
 
+message ClusterScan {
+    string cursor = 1;
+    optional string match_pattern = 2;
+    optional int64 count = 3;
+    optional ObjectType object_type = 4;
+}
```

Comment:
All this could be supplied as command arguments

Reply:
It is not sent to redis-rs as a command. redis-rs use it as part of a big logic that's ends up as in a command which will be generated at some later point in the logic.
So we dont want to send it as a command.
And we don't want to block the option of the user to send normal SCAN by sending it as a command and catch it on the way.